numpy/_core/defchararray.py

```diff
@@ -158,6 +158,13 @@ def not_equal(x1, x2):
     See Also
     --------
     equal, greater_equal, less_equal, greater, less
+
+    Examples
+    --------
+    >>> x1 = np.array(['a', 'b', 'c'])
+    >>> np.char.not_equal(x1, 'b')
+    array([True, False,  True])
```

Comment:
Please adjust all of the outputs to exactly match what the REPL would produce. Maybe it'd be best to copy-paste the statements into the REPL and just transfer over the result to the doc.

Reply:
This is what I am getting:
```python
>>> import numpy as np
>>> x1 = np.array(['a', 'b', 'c'])
>>> np.char.not_equal(x1, 'b')
array([ True, False,  True])
```

Reply:
Am I overlooking some detail here or what do you mean?

Reply:
I See you are talking about the spacing!

Reply:
Ditto for all the others as well.